python/sglang/srt/managers/tp_worker.py

```diff
@@ -667,7 +668,6 @@ def forward_decode_batch(self, batch: ScheduleBatch):
                 return
 
         # Update batch tensors
-        self.decode_forward_ct = (self.decode_forward_ct + 1) % (1 << 30)
```

Comment:
it is also used somewhere else https://github.com/sgl-project/sglang/blob/726d34783034369b0c8453565bd830d57491bc32/python/sglang/srt/managers/tp_worker.py#L723

Reply:
I did NOT remove this. I just moved it into the place where the decode is actually done. Also, I removed the overflow avoidance code since python int are big int and never overflows.

Reply:
sorry, I miseed that change. Please move the code back here as we want to keep `def forward_step(self):` as short as possible. It is the highest level of scheduling so we want to make it short and easy to understand.